armi/cases/tests/test_cases.py

```diff
@@ -89,7 +89,11 @@ def test_summarizeDesign(self):
             case = cases.Case(cs)
             c2 = case.clone()
             c2.summarizeDesign(True, True)
-            self.assertTrue(os.path.exists("Core Design Report.html"))
+            self.assertTrue(
+                os.path.exists(
```

Comment:
We don't want to test the file contents at all? Look for a set string we added, or test the length of the file contents, to show it isn't empty?